content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,607 @@
+
+To update an existing Bevy App or Plugin to **Bevy 0.10**, check out our [0.9 to 0.10 Migration Guide](/learn/book/migration-guides/0.9-0.10/).
+
+Since our last release a few months ago we've added a _ton_ of new features, bug fixes, and quality of life tweaks, but here are some of the highlights:
+
+<!-- more -->
+
+* **ECS Schedule v3**: Bevy now has much simpler, more flexible scheduling. Systems are now stored in a unified schedule, commands can be applied explicitly via `apply_system_buffers`, and a whole lot of quality of life and bug fixes.
+
+## ECS Schedule v3: simpler, more flexible scheduling
+
+<div class="release-feature-authors">authors: @alice-i-cecile, @maniwani, @WrongShoe, @cart, @jakobhellermann, @JoJoJet, @geieredgar and a whole lot more </div>
+
+Thanks to the fantastic work of our ECS team, the hotly awaited ["stageless" scheduling RFC](https://github.com/bevyengine/rfcs/blob/main/rfcs/45-stageless.md) has been implemented!
+
+Schedule v3 is the culmination of significant design and implementation work. Scheduling APIs are a central and defining part of the Bevy developer experience, so we had to be very thoughtful and meticulous about this next evolution of the API. In addition to the [RFC PR](https://github.com/bevyengine/rfcs/pull/45), the [initial implementation PR](https://github.com/bevyengine/bevy/pull/6587) by `@maniwani` and the [Bevy Engine internals port PR](https://github.com/bevyengine/bevy/pull/7267) by `@alice-i-cecile` are great places to start if you would like a view into our process and rationale. As we all know, plans and implementations are two different things. Our final implementation is a bit different from the initial RFC (in a good way).
+
+There's been a lot of changes, but we've put a lot of care into ensuring the [migration path](/learn/book/migration-guides/0.9-0.10/) for existing applications is relatively straightforward. Don't sweat it!
+
+Lets take a look at what shipped in 0.10!
+
+## A Single Unified Schedule
+
+Have you ever wanted to specify that `system_a` runs before `system_b`, only to be met with confusing warnings that `system_b` isn't found because it's in a different stage?
+
+No more! All systems within a single **schedule** are now stored in a single data structure with a global awareness of what's going on.
+
+This simplifies our internal logic, makes your code more robust to refactoring, and allows plugin authors to specify high-level invariants (e.g. "movement must occur before collision checking") without locking themselves in to an exact schedule location.
+
+![main_schedule_diagram](main_schedule_diagram.svg)
```

Comment:
It's not merged yet, but you might want to use the color coded version (which also includes de a11y related systems): https://github.com/jakobhellermann/bevy_mod_debugdump/pull/16

https://user-images.githubusercontent.com/188612/222808455-bf51e4e4-6e47-460c-82ac-264516ba37b2.svg
![image](https://user-images.githubusercontent.com/188612/222842007-6e4954a3-549a-44fa-a56d-da0cb84c6eb6.png)

Reply:
Much nicer!

Reply:
Ooh thats really nice!